tests/lib/rules/toggle-button-needs-labelling-v9.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/toggle-button-needs-labelling-v9"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+ruleTester.run("toggle-button-needs-labelling-v9", rule, {
+    valid: [
+        '<ToggleButton aria-label="Toggle Example" />',
```

Comment:
Let's also make sure our test cases cover the situation with the icon prop.

correct
  
```
  <ToggleButton >
    Example
  </ToggleButton>
  
   <ToggleButton
    icon={<CalendarMonthRegular />}
  >
    Example
  </ToggleButton>
   
       <Tooltip content="With calendar icon only" relationship="label">
        <ToggleButton icon={<CalendarMonthRegular />} />
      </Tooltip>
	  
  <ToggleButton
    icon={<CalendarMonthRegular />}
    aria-label="Monthly calendar"
  >
  </ToggleButton>
	    
    <Label id="calendar-1">Compound example</Label>
    <ToggleButton aria-labelledby="calendar-1" icon={<CalendarMonthRegular />} />
```
  
  incorrect
  
```
    <ToggleButton></ToggleButton>
  
  
    <ToggleButton icon={<CalendarMonthRegular />}></ToggleButton>
	
	
	<Label>Compound example</Label>
    <ToggleButton icon={<CalendarMonthRegular />} />
```